### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by sasays** [kimai/kimai#2386] [Questions & Answers]
Hi @kevinpapst 

I have just download the zip file of your source code
and pasted the files in Xampp-->htdocs folder in local machine

edited the **.env** file and added mysql connection 

after running the this in Xampp server im getting this error 
![image](https://user-images.githubusercontent.com/79901539/109619595-f99eac80-7b5e-11eb-9ab6-1e540cb32dab.png)

can you help ..!

**Comment by kevinpapst** [comment]
Follow the installation documentation, then switch to PHP 7 and then lets talk again
